wpeverest/everest-forms
# Feature - Email scheduling in daily, weekly or monthly for the progress data of form

### All Submissions:



### Changes proposed in this Pull Request:

Previously, there was no option to send the email in the routine basis(either daily, or monthly or weekly) regarding the form data entries. This PR adds the feature to schedule the email on routine basis to send the statistics report regarding the form entries from the desired forms.

Closes # .

### How to test the changes in this Pull Request:

1. Go to EVF Global Settings > Reporting. 
2. Click on the checkbox to enable the routine statistics report.
3. Select the desired Email Frequency(Note: Day is allowed to choose only if Weekly is selected from the Email frequency dropdown).
4. Enter the email to send and the subject
5. Now, you can select the forms whose data must be emailed on routine basis.
6. Click on Save button.
7. Finally, verify whether proper email is being sent or not according to the schedule type.

### Types of changes:



### Other information:


### Changelog entry

> Feature - Email scheduling in daily, weekly or monthly for the progress data of form
